crown-court-proceeding/src/test/java/uk/gov/justice/laa/crime/crowncourt/controller/CrownCourtProceedingIntegrationTest.java

```diff
@@ -202,6 +207,78 @@ void givenAValidContent_whenUpdateApplicationIsInvoked_thenUpdateApplicationIsSu
         assertThat(result.getResponse().getContentAsString()).isEqualTo(objectMapper.writeValueAsString(updateApplicationResponse));
     }
 
+
+    @Test
+    void givenAEmptyOAuthToken_whenUpdateIsInvoked_thenFailsUnauthorizedAccess() throws Exception {
+        mvc.perform(RequestBuilderUtils.buildRequestGivenContent(
+                        HttpMethod.PUT, "{}", ENDPOINT_URL + UPDATE_CC_URL, Boolean.FALSE))
+                .andExpect(status().isUnauthorized());
+    }
+
+    @Test
+    void givenAInvalidContent_whenUpdateIsInvoked_thenFailsBadRequest() throws Exception {
+        mvc.perform(RequestBuilderUtils.buildRequestGivenContent(
+                        HttpMethod.PUT, objectMapper.writeValueAsString(
+                                TestModelDataBuilder.getApiUpdateApplicationRequest(Boolean.FALSE)), ENDPOINT_URL + UPDATE_CC_URL))
+                .andExpect(status().isBadRequest());
+    }
+
+    @Test
+    void givenAValidContent_whenUpdateIsInvoked_thenUpdateIsSuccess() throws Exception {
+
+        ApiUpdateApplicationRequest apiUpdateApplicationRequest = TestModelDataBuilder.getApiUpdateApplicationRequest(Boolean.TRUE);
+        apiUpdateApplicationRequest.setCaseType(CaseType.APPEAL_CC);
+        stubForUpdateCC();
+        mvc.perform(RequestBuilderUtils.buildRequestGivenContent(
+                        HttpMethod.PUT, objectMapper.writeValueAsString(apiUpdateApplicationRequest), ENDPOINT_URL + UPDATE_CC_URL))
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$.crownCourtSummary.repOrderDecision").value("Granted - Passed Means Test"))
+                .andExpect(jsonPath("$.crownCourtSummary.repType").value("Crown Court Only"))
+                .andExpect(jsonPath("$.crownCourtSummary.evidenceFeeLevel").value("LEVEL1"))
+                .andExpect(jsonPath("$.crownCourtSummary.repOrderCrownCourtOutcome[0].outcome").value("SUCCESSFUL"))
```

Comment:
Can these not use the existing Enums here?

Reply:
Thanks. Done now